openmmml/models/anipotential.py

```diff
@@ -29,6 +29,7 @@
 USE OR OTHER DEALINGS IN THE SOFTWARE.
 """
 
+from turtle import pos, position
```

Comment:
What is this for?

Reply:
Unused, not sure how that got there. I'll remove it.